azure-pipelines-windows.yml

```diff
@@ -6,6 +6,14 @@ jobs:
   pool:
     vmImage: 'vs2017-win2016'
   steps:
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core 2'
+    inputs:
+      version: 2.x
```

Comment:
There is more than one PR with this same changes.

Reply:
That was my point to remove the diff. PR https://github.com/zkSNACKs/WalletWasabi/pull/2363 will be the ultimate solution, so this is just temporary.